manifests/a/Amazon/WorkspacesClient/5.2.1/Amazon.WorkspacesClient.yaml

```diff
@@ -1,8 +1,8 @@
-# Automatically updated by the winget bot at 2022/Aug/03
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
-
-PackageIdentifier: Amazon.WorkspacesClient
-PackageVersion: 5.2.0.3089
-DefaultLocale: en-US
-ManifestType: version
-ManifestVersion: 1.1.0
+# Created with YamlCreate.ps1 v2.1.3 $debug=AUSU.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Amazon.WorkspacesClient
+PackageVersion: 5.2.1
```

Comment:
Version should be: 5.2.1.3123

Reply:
No. The marketed version is 5.2.1, the DisplayVersion is 5.2.1.3123, which is shown in the AppsAndFeaturesEntries